wurst/objects/items/ScrollsDefinition.wurst

```diff
@@ -0,0 +1,421 @@
+    return createUsableItem(ITEM_SCROLL_LIVING_DEAD)
+        ..setCooldownGroup(ABILITY_SKELETAL_MINION.toRawCode())
+        ..setLumberCost(7)
+        ..setAbilities(ABILITY_SKELETAL_MINION.toRawCode())
+        ..setStockReplenishInterval(120)
+        ..setModelUsed(LocalItems.scrollYellow)
+        ..setInterfaceIcon(Icons.bTNSnazzyScroll)
+        ..setNameEnhance("Scroll of Living Dead")
+        ..setTooltipExtended(LD_TT)
+
+@compiletime function createStoneShieldScroll() returns ItemDefinition
+    return createUsableItem(ITEM_SCROLL_STONE_ARMOR)
+        ..setCooldownGroup(ABILITY_FROST_ARMOR.toRawCode())
+        ..setLumberCost(9)
+        ..setAbilities(ABILITY_FROST_ARMOR.toRawCode())
+        ..setStockReplenishInterval(120)
+        ..setModelUsed(LocalItems.scrollOrange)
+        ..setInterfaceIcon(Icons.bTNScrollUber)
+        ..setNameEnhance("Scroll of Stone Shield")
+        ..setTooltipExtended(SS_TT)
+
+@compiletime function createHasteScroll() returns ItemDefinition
+    return createUsableItem(ITEM_SCROLL_HASTE)
+        ..setCooldownGroup(ABILITY_HASTE_AOE.toRawCode())
+        ..setLumberCost(10)
+        ..setAbilities(ABILITY_HASTE_AOE.toRawCode())
+        ..setStockReplenishInterval(120)
+        ..setModelUsed(LocalItems.scrollHaste)
+        ..setInterfaceIcon(Icons.bTNScrollOfHaste)
+        ..setNameEnhance("Scroll of Haste")
+        ..setTooltipExtended(SH_TT)
+
+@compiletime function createTsunamiScroll() returns ItemDefinition
+    return createUsableItem(ITEM_SCROLL_TSUNAMI)
+        ..setCooldownGroup(ABILITY_TSUNAMI.toRawCode())
+        ..setLumberCost(9)
+        ..setAbilities(ABILITY_TSUNAMI.toRawCode())
+        ..setStockReplenishInterval(120)
+        ..setModelUsed(LocalItems.scrollPurple)
+        ..setInterfaceIcon(Icons.bTNSnazzyScrollPurple)
+        ..setNameEnhance("Scroll of Tsunami")
+        ..setTooltipExtended(ST_TT)
+
+@compiletime function createCycloneScroll() returns ItemDefinition
+    return createUsableItem(ITEM_SCROLL_CYCLONE)
+        ..setCooldownGroup(ABILITY_CYCLONE.toRawCode())
+        ..setLumberCost(9)
+        ..setAbilities(ABILITY_CYCLONE.toRawCode())
+        ..setStockReplenishInterval(120)
+        ..setModelUsed(LocalItems.scrollCyan)
+        ..setInterfaceIcon(Icons.bTNBansheeMaster)
+        ..setNameEnhance("Scroll of Cyclone")
+        ..setTooltipExtended(SC_TT)
+
+
+let BUFF_CYCLONE = 'Bcyc'
+let BUFF_CYCLONE_EXTRA = 'Bcy2'
+
+function onCyclone(unit target)
+    target.addAbility(GHOST_VIS_ID)
+    doPeriodicallyTimed(0.1, 15) buffPoller ->
+        if not target.hasAbility(BUFF_CYCLONE) and not target.hasAbility(BUFF_CYCLONE_EXTRA)
+            target.removeAbility(GHOST_VIS_ID)
+            buffPoller.stop()
+
+function unit.isFire() returns bool
+    return this.getTypeId() == UNIT_FIRE
+        or this.getTypeId() == UNIT_MAGE_FIRE_SUMMONED
+        or this.getTypeId() == UNIT_MAGE_FIRE
+
+function onTsunami(unit caster, vec2 target)
```

Comment:
I thought you were using 2 cone abilities tied to a single dummy spell for this. Did you decide not to?

Reply:
I decided not to, didn't work at my first try so I chose the lazy path and let it this way.